client/app/pages/queries/view.js

```diff
@@ -249,6 +249,9 @@ function QueryViewCtrl(
       $scope.saveQuery(options, data);
     }
 
+    request.options = extend({}, request.options);
+    request.options.parameters = map(request.options.parameters, param => omit(param, 'urlPrefix'));
```

Comment:
Won't this add parameters where it wasn't there before?

Reply:
@arikfr Added new commit - with another approach